paddle/fluid/operators/math/prelu.cu

```diff
@@ -18,120 +18,88 @@ namespace paddle {
 namespace operators {
 namespace math {
 
-static const int CUDA_NUM_THREADS = 1024;
-static const int CUDA_MAX_NUM_BLOCKS = 65535;
-inline static int GET_NUM_BLOCKS(const int N) {
-  return (N + CUDA_NUM_THREADS - 1) / CUDA_NUM_THREADS;
+static constexpr int PD_CUDA_NUM_THREADS = 1024;
```

Comment:
according to the code style of paddle, static variable should be declared in s_xxx format. In this condition, “#define CUDA_NUM_THREADS 1024” looks better

Reply:
Have you check whether there is a global variable to define this `CUDA_NUM_THREADS`，if there is, you'd better to use the global one for consistence . if not, `CUDA_NUM_THREADS` becomes a local configuration, which is not proper for `PD_` or `PADDLE_` prefix.

Reply:
Done. And there is a global variable called PADDLE_CUDA_NUM_THREADS which is set 512. But i think 1024 is better value, so I use the #define instructive statement to set the value to 1024.